vrep/constrained_kinematic_control/constrained_kinematic_control.m

```diff
@@ -0,0 +1,125 @@
+% (C) Copyright 2023 DQ Robotics Developers
+% This file is part of DQ Robotics.
+%
+%     DQ Robotics is free software: you can redistribute it and/or modify
+%     it under the terms of the GNU Lesser General Public License as published 
+%     by the Free Software Foundation, either version 3 of the License, or
+%     (at your option) any later version.
+%
+%     DQ Robotics is distributed in the hope that it will be useful,
+%     but WITHOUT ANY WARRANTY; without even the implied warranty of
+%     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+%     GNU Lesser General Public License for more details.
+%
+%     You should have received a copy of the GNU Lesser General Public License
+%     along with DQ Robotics.  If not, see <http://www.gnu.org/licenses/>.
+%
+% DQ Robotics website: dqrobotics.github.io
+%
+% Contributors to this file:
+%     Juan Jose Quiroz Omana (juanjqo@g.ecc.u-tokyo.ac.jp)
+
+clear all;
+close all;
+clc;
+
+include_namespace_dq
+
+vi = DQ_VrepInterface();
+vi.disconnect_all();
+vi.connect('127.0.0.1',19997);
+vi.set_synchronous(true);
+vi.start_simulation();
+
+try
+
+jointnames={'LBR4p_joint1','LBR4p_joint2',...
```

Comment:
Lines not corresponding to set up and do not depend on CoppeliaSim should be moved outside the `try-catch` block. I suggest moving 36--69.